centreon/www/include/configuration/configGenerate/xml/moveFiles.php

```diff
@@ -334,6 +336,25 @@
                         }
                     }
                 }
+                /**
+                 * VMWare configuration
+                 */
+                if (count($listVmWareFile) > 0) {
+                    foreach ($listVmWareFile as $fileCfg) {
+                        if (! copy($fileCfg, _CENTREON_ETC_ . '/' . 'centreon_vmware.json')) {
+                            throw new Exception(sprintf(
+                                <<<'MSG'
+                                    Could not write to VMWare's configuration file '%s' for monitoring server '%s'.
+                                    Please add writing permissions for the webserver's user.
+                                    MSG,
+                                basename($fileCfg),
+                                $host['name']
+                            ));
+                        } elseif (posix_getuid() === fileowner(_CENTREON_ETC_ . '/' . 'centreon_vmware.json')) {
+                            @chmod(_CENTREON_ETC_ . '/' . 'centreon_vmware.json', 0664);
```

Comment:
It's not a good practice to use @ before the native method of PHP. And by security we don't use chmod in the code ;) You should check with the security team. And if chmod failed, what do you want to do ?

Reply:
Same behaviour than other files, we do nothing and an error is prompt in the UI

Reply:
ok for me